src/plugins/yy/yyplugin.cpp

```diff
@@ -307,9 +335,22 @@ struct Context
         return name;
     }
 
+    const QString &resourceId(const QString &filePath)
+    {
+        if (filePath.isEmpty())
+            return "";
```

Comment:
Oh, that check already sneaked in, I thought I excluded those lines. Sorry about the confusion. But yes, I am aware and will fix it.
I *think* it happens if an image layer has no image defined. I just grabbed a Tile file from someone else for testing this and it would write some empty spriteIDs regardless.

Reply:
Done.